core/txpool/txpool.go

```diff
@@ -44,14 +46,12 @@ const (
 	TxStatusIncluded
 )
 
-var (
-	// reservationsGaugeName is the prefix of a per-subpool address reservation
-	// metric.
-	//
-	// This is mostly a sanity metric to ensure there's no bug that would make
-	// some subpool hog all the reservations due to mis-accounting.
-	reservationsGaugeName = "txpool/reservations"
-)
+// reservationsGaugeName is the prefix of a per-subpool address reservation
+// metric.
+//
+// This is mostly a sanity metric to ensure there's no bug that would make
+// some subpool hog all the reservations due to mis-accounting.
+var reservationsGaugeName = "txpool/reservations"
```

Comment:
I'll revert this diff to upsteam... my linter automatically applies this on save.